src/sqore.ts

```diff
@@ -67,6 +67,13 @@ export class Sqore {
 
         // Render operations at starting at given depth
         circuit.operations = this.selectOpsAtDepth(circuit.operations, renderDepth);
+
+        // If only one top-level operation, expand automatically:
+        if (circuit.operations.length == 1 && circuit.operations[0].dataAttributes != null) {
```

Comment:
Should we also check for the existence of `id` in `dataAttributes`?
```suggestion
        if (circuit.operations.length == 1 && circuit.operations[0].dataAttributes != null && circuit.operations[0].dataAttributes.hasOwnProperty('id')) {
```